deployments/charts/kuma/values.yaml

```diff
@@ -60,6 +60,11 @@ controlPlane:
   # secrets: {}
 
   envVars: {}
+
+#  config: |
+#    xdsServer:
+#      dataplaneStatusFlushInterval: 2s
```

Comment:
nit: Maybe have a more generic config as an example here. Something about changing a port

Reply:
I don't think changing port is a common case on Kubernetes, because you have network separation within the container. I felt like this one is a real use case because you can actually tune the performance of the control plane with this (however, I made a mistake, this should be `dataplaneStatusFlushInterval`)